### ISSUE TITLE
[Obsolete] 'connective tissue disease'

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#3570]
**Mondo term (ID and Label)**
'connective tissue disease'

**Reason for deprecation**
The classifications under this term do not entirely make sense. It is recommended by Peter and Ada that we obsolete this term and reclassify the children elsewhere

**Suggested term to consider**

**Comment by sabrinatoro** [comment]
part of #3155

**Comment by nicolevasilevsky** [comment]
Per the Mondo call today, we decided that we probably want to keep Connective tissue disease as it is being used by EFO.
We propose to keep this and remove the logical definition and add a comment that says: This excludes diseases that are classically considered connective tissues like bone tissue diseases.

Removing the logical def will no longer cause the automatic classification of diseases like bone disease as a child of connective tissue disorder.

This will be excluded from the Harrison view.

cc @cmungall @dosumis @sabrinatoro @ahamosh @pnrobinson

**Comment by pnrobinson** [comment]
There are many medical errors in the EFO hierarchy

https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_1001986

errors in the sense that essentially any disease can be a connective tissue disease (e.g., heart attach, because the heart contains connective tissue), but it makes no sense at all to classify them that way. I think we should feed our discussions back to EFO and see if they can be convinced to alter their hierachy?

**Comment by ahamosh** [comment]
I will leave this battle to you all, but I really think we need to clean all of this up.  If the ontologies are what the AI and machine learning will use, they need to be correct so they can learn. Probably the same applies to humans...

**Comment by dosumis** [comment]
I agree that as a logical grouping it doesn't make much sense, but it's clearly a much used grouping, presumably held together by convention as much as anything:

https://my.clevelandclinic.org/health/diseases/14803-connective-tissue-diseases
https://www.mayoclinic.org/diseases-conditions/mixed-connective-tissue-disease/symptoms-causes/syc-20375147
https://www.webmd.com/a-to-z-guides/connective-tissue-disease
https://www.cedars-sinai.org/health-library/diseases-and-conditions/c/connective-tissue-disorders.html

 Doesn't it at least provide a familiar entry point for clinicians?  

Given that we can't populate this logically, my main concern is whether we can manually maintain a sufficiently complete set of subclasses.  If so, I'm happy to keep it - with appropriate public and editor facing comments to make the nature of the term clear.

Note - whatever Mondo does, EFO will continue to need and use this class (with an EFO ID if necessary) and other grouping classes like it as they are required by major users - particularly OTAR.  Any annotations to specific diseases in this branch of EFO will use Mondo IDs, so are perfectly amenable to use in analysis excluding EFO and using the Mondo hierarchy alone.

CC @zoependlington

**Comment by pnrobinson** [comment]
@dosumis I was not trying to say that the phrase connective tissue disease is wrong. I was saying that many of classes that EFO has underneath this term would not be considered to be connective tissue diseases by physicians. Therefore, searches or AI analysis using EFO will return unexpected/undesired results for medical users.

**Comment by dosumis** [comment]
> @dosumis I was not trying to say that the phrase connective tissue disease is wrong. I was saying that many of classes that EFO has underneath this term would not be considered to be connective tissue diseases by physicians. Therefore, searches or AI analysis using EFO will return unexpected/undesired results for medical users.

Got it.  That is useful to feed back.  I wonder how much of the error comes inference from the Mondo logical def we are now removing.  This is pulling in diseases of anatomical tissues that are considered connective tissue (and so are classified as such by Uberon) even if,  conventionally, these diseases are not classified as connective tissue diseases.

**Comment by nicolevasilevsky** [comment]
@sabrinatoro do you want to work on this? I think we should just close my other PR and separate out these issues.

- [ ] remove the logical definition from 'connective tissue disorder' and add a comment that says: This excludes diseases that are classically considered connective tissues like bone tissue diseases.
- [ ] on a separate PR, I think we still want to obsolete MONDO_0020012 'systemic or rheumatic disease' and MONDO_0017133 'genetic systemic or rheumatologic disease', but we should hold off on this until Nov or Dec.

and this is related to https://github.com/monarch-initiative/mondo/issues/3606, which will add a new child named 'Mendelian connective tissue disorder'

We should ask folks to review the revisions before we commit (you could paste in screenshot of the hierarchy)

**Comment by dosumis** [comment]
Suggested comment:  "Following clinical usage of the term 'connective tissue disorder' this grouping excludes some diseases affecting tissues classically considered to be connective tissue, such as bone."
